exercises/allergies/src/example.h

```diff
@@ -0,0 +1,27 @@
+#ifndef _ALLERGIES_H
+#define _ALLERGIES_H
+
+#include <stdbool.h>
+
+typedef enum {
+   Allergen_First = 0,
+   Allergen_Eggs = Allergen_First,
+   Allergen_Peanuts,
+   Allergen_Shellfish,
+   Allergen_Strawberries,
+   Allergen_Tomatoes,
+   Allergen_Chocolate,
+   Allergen_Pollen,
+   Allergen_Cats,
+   Allergen_Last = Allergen_Cats
+} Allergen_t;
+
+typedef struct {
+   int count;
+   Allergen_t *allergens;
```

Comment:
Given that we know exactly how many allergens there are prior to declaring this type, why wouldn't we simply create a properly sized array for `allergens` instead of requiring a memory allocation?